packages/apollos-ui-kit/src/inputs/Radio/index.js

```diff
@@ -1,66 +1,3 @@
-import React, { Component } from 'react';
-import { View } from 'react-native';
-import PropTypes from 'prop-types';
-
-import RadioButton from './RadioButton';
-
-export default class Radio extends Component {
-  static Button = RadioButton;
-
-  static propTypes = {
-    children: PropTypes.node,
-    onChange: PropTypes.func,
-    value: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
-    style: PropTypes.any, // eslint-disable-line
-  };
-
-  static childContextTypes = {
-    onSelectValue: PropTypes.func,
-    currentValue: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
-  };
-
-  static defaultProps = {
-    children: null,
-    onChange() {},
-    value: null,
-  };
-
-  state = {
-    value: this.props.value,
-  };
-
-  getChildContext() {
-    return {
-      onSelectValue: this.selectValue,
-      currentValue: this.state.value,
-    };
-  }
-
-  componentWillReceiveProps(nextProps) {
-    if (nextProps.value !== this.props.value) {
-      this.setState(
-        {
-          value: nextProps.value,
-        },
-        this.notifyValueChanged
-      );
-    }
-  }
-
-  selectValue = (value) => {
-    this.setState(
-      {
-        value,
-      },
-      this.notifyValueChanged
-    );
-  };
-
-  notifyValueChanged = () => {
-    this.props.onChange(this.state.value);
-  };
-
-  render() {
-    return <View style={this.props.style}>{this.props.children}</View>;
-  }
-}
+export { default } from './Radio';
```

Comment:
`Radio` should probably be the default export from `/Radio`. Test this to make sure it works.
```suggestion
export default from './Radio';
```